pkg/model/database/database.go

```diff
@@ -104,26 +105,29 @@ var ProjectFields = struct {
 // Useful in GORM .Order() statements to order the results based on a specific
 // column, which does not support the regular Go field names.
 var ProjectColumns = struct {
-	ProjectID   string
-	Name        string
-	GroupName   string
-	Description string
-	TokenID     string
-	GitURL      string
+	ProjectID       string
+	RemoteProjectID string
+	Name            string
+	GroupName       string
+	Description     string
+	TokenID         string
+	GitURL          string
 }{
-	ProjectID:   "project_id",
-	Name:        "name",
-	GroupName:   "group_name",
-	Description: "description",
-	TokenID:     "token_id",
-	GitURL:      "git_url",
+	ProjectID:       "project_id",
+	RemoteProjectID: "remote_project_id",
+	Name:            "name",
+	GroupName:       "group_name",
+	Description:     "description",
+	TokenID:         "token_id",
+	GitURL:          "git_url",
 }
 
 // Project holds data about an imported project. A lot of the data is expected
 // to be populated with data from the remote provider, such as the description
 // and avatar.
 type Project struct {
 	ProjectID       uint      `gorm:"primaryKey"`
+	RemoteProjectID string    `gorm:"not null;default:''"`
```

Comment:
Think it'd be wise to add an index to this column.

```suggestion
	RemoteProjectID string    `gorm:"not null;default:'';index:project_idx_remote_project_id"`
```

Reply:
Wait, no, we won't be querying on this field. Ignore this